README.md

```diff
@@ -168,21 +168,20 @@ CA Cluster Issuer Sample `values.yaml`
 
 Harbor is an open source trusted cloud native registry project that stores, signs, and scans content. Harbor extends the open source Docker Distribution by adding the functionalities usually required by users such as security, identity and management. Having a registry closer to the build and run environment can improve the image transfer efficiency. Harbor supports replication of images between registries, and also offers advanced security features such as user management, access control and activity auditing.
 
-- The [contour package](#contour) is a prerequisite for Harbor, so that must be installed
-  first.
+- The [contour package](#contour) is a prerequisite for Harbor v2.9.x and v2.8.x, so that must be installed
+  first if you are using these versions.
+- Harbor v2.11.2 supports exposing the registry using loadbalancer, and contour is not required.
 - Follow the instructions under [Installing and Configuring Harbor on a Supervisor](https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-with-tanzu-services-workloads/GUID-8C645717-C473-4E67-8DEE-049B71447B86.html).
 
 ### Harbor Versions
 
-- Download latest version: [Harbor v2.9.1](https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=supervisor-services&path=harbor/v2.9.1/harbor.yml)
-- Download version: [Harbor v2.8.2](https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=supervisor-services&path=harbor/v2.8.2/harbor.yml)
```

Comment:
Are you dropping the list of older release versions for a reason? I noticed the text above still mentions v2.8.x (requires Contour)

Reply:
Earlier we decided to keep only 2.9.x and 2.11.x b/c we've stopped maintaining Harbor v2.8.x OSS for some time.  I should also remove the reference to v2.8.x from "requires Contour" paragraph